src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/ExtendedSplashScreen.Windows.crossruntime.cs

```diff
@@ -255,7 +255,10 @@ private static async Task<string> ApplicationPathFromFileName(Assembly? entry, s
 #pragma warning restore SYSLIB0044 // Type or member is obsolete
 			if (codebase is not null)
 			{
-				var manifestPath = Path.Combine(Path.GetDirectoryName(codebase) ?? string.Empty, fileName);
+				UriBuilder uri = new UriBuilder(codebase);
+				string path = Uri.UnescapeDataString(uri.Path);
```

Comment:
Let's add a comment here explaining the reason for this call. Mentioning the "file://' prefix that was causing problems